copilot-cli/skills/synthesis/SKILL.md

```
@@ -127,6 +127,73 @@ EOF
+At the start of synthesis, check the workflow platform:
+
+```powershell
+~/.copilot/scripts/workflow-state.ps1 get docs/workflow-state/<feature>.state.json '.platform'
+```
+
+- If platform is `"azure-devops"` or `"ado"`: Use Step 5a/5b below
+- If platform is `"github"` or not set: Use Step 5 above (gh CLI)
+
+### Step 5a: Create Pull Request (ADO)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)